luggage.make

```diff
@@ -204,6 +205,10 @@ enlprojdir: resourcedir
 scratchdir:
 	@sudo mkdir -p ${SCRATCH_D}
 
+outputdir:
+	@sudo mkdir -p ${OUTPUT_D}
```

Comment:
Only do these if OUTPUT_D is not '.'; e.g. prefix this line and next with `[[ ${OUTPUT_D} == "." ]] || `